Packs/AbnormalSecurity/Integrations/AbnormalSecurity/AbnormalSecurity.yml

```diff
@@ -276,6 +361,123 @@ script:
     - contextPath: AbnormalSecurity.AbnormalCaseDetails.firstObserved
       description: First time suspicious behavior was observed.
       type: String
+  - arguments:
+    - description: A UUID representing the abuse campaign id. Can be Can be retrieved by first running command to list abuse mailbox campaigns
+      isArray: false
+      name: campaign_id
+      required: true
+    - description: Returns test data if set to `True`
+      isArray: false
+      name: mock-data
+      required: false
+    - description: Subtenant of the user (if applicable)
+      isArray: false
+      name: subtenant
+      required: false
+    description: Get details of an Abuse Mailbox campaign
+    name: abnormal-security-get-abuse-mailbox-campaign
+    outputs:
+      - contextPath: AbnormalSecurity.AbuseCampaignDetails.campaignId
```

Comment:
Seems that after the changes, you didn't update the yml outputs.
Please fix according to the new one.